documentation/Bundling.md

```diff
@@ -179,16 +179,46 @@ Once this is done, you can use rollup by configuring your project in the way tha
 
 In order to use Azure SDK libraries inside JS, you need to import code from the package you installed earlier. Create `src/index.js` with the following content:
 
+```js
+// src/index.js
+import { SomeClient } from "@azure/some-sdk-package";
+// Now do something interesting with the client
+```
+
+Next we need to configure Rollup to take the above code and turn it into a bundle. Save the following `rollup.config.mjs` file next to your `package.json` file you created earlier:
+
+```js
+// rollup.config.mjs
```

Comment:
it seems we still have issues with js part: https://github.com/Azure/azure-sdk-for-js/issues/32430#issuecomment-2574277414, also i think we would update the sample project together if successful: https://github.com/Azure/azure-sdk-for-js/tree/main/samples/Bundling/rollup?

Reply:
Good point @MaryGao! I updated the bundling samples.